bip-0341.mediawiki

```diff
@@ -175,6 +176,8 @@ The parity bit will be required for spending the output with a script path.
 In order to allow spending with the key path, we define <code>taproot_tweak_seckey</code> to compute the secret key for a tweaked public key.
 For any byte string <code>h</code> it holds that <code>taproot_tweak_pubkey(pubkey_gen(seckey), h)[1] == pubkey_gen(taproot_tweak_seckey(seckey, h))</code>.
 
+Note that the secret key must be negated, if necessary, to produce an even-y pubkey before applying the tweak.
```

Comment:
This could be misunderstood as an instruction to negate the secret key before calling `taproot_tweak_seckey`. Also, public keys are 32 byte arrays and therefore don't have even- or odd-Y (and the corresponding points are always even-Y).

How about something like this:
"Note that because tweaks are applied to 32-byte public keys, <code>taproot_tweak_seckey</code> may need to negate the secret key before applying the tweak."

Reply:
Thank you. Changed the wording.